community-membership.md

```diff
@@ -0,0 +1,168 @@
+- May either self-nominate, be nominated by an approver in this subproject, or be nominated by a robot
+
+### Responsibilities and privileges
+
+The following apply to the part of codebase for which one would be a reviewer in
+an [CODEOWNERS] file (for repos using the bot).
+
+- Tests are automatically run for PullRequests from members of the Sigstore GitHub organization
+- Code reviewer status may be a precondition to accepting large code contributions
+- Responsible for project quality control via [code reviews]
+  - Focus on code quality and correctness, including testing and factoring
+  - May also review for more holistic issues, but not a requirement
+- Expected to be responsive to review requests as per [community expectations]
+- Assigned PRs to review related to the project of expertise
+- Assigned test bugs related to the project of expertise
+- May get a badge on PR and issue comments
+
+## Approver
+
+Code approvers are able to both review and approve code contributions.  While
+code review is focused on code quality and correctness, approval is focused on
+holistic acceptance of a contribution including: backwards / forwards
+compatibility, adhering to API and flag conventions, subtle performance and
+correctness issues, interactions with other parts of the system, etc.
+
+**Defined by:** *approvers* entry in an CODEOWNERS file in a repo owned by the
+Sigstore project.
+
+Approver status is scoped to a part of the codebase.
+
+### Requirements
+
+The following apply to the part of codebase for which one would be an approver
+in an [CODEOWNERS] file (for repos using the bot).
+
+- Reviewer of the codebase for at least 3 months
+- Primary reviewer for at least 10 substantial PRs to the codebase
+- Reviewed or merged at least 30 PRs to the codebase
+- Nominated by a subproject owner
+  - With no objections from other subproject owners
+  - Done through PR to update the top-level CODEOWNERS file
+
+### Responsibilities and privileges
+
+The following apply to the part of codebase for which one would be an approver
+in an [CODEOWNERS] file (for repos using the bot).
+
+- Approver status may be a precondition to accepting large code contributions
+- Demonstrate sound technical judgement
+- Responsible for project quality control via [code reviews]
+  - Focus on holistic acceptance of contribution such as dependencies with other features, backwards / forwards
+    compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests as per [community expectations]
+- Mentor contributors and reviewers
+- May approve code contributions for acceptance
+
+## Inactive members
+
+_Members are continuously active contributors in the community._
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+_TBD how to track inactive members and phase down._
```

Comment:
Maybe some guidelines:

* Lack of significant contribution to the repository
* Not assisting with reviews or triaging issues